### PR TITLE
(BIDS-2548) add total deposits columns

### PR DESCRIPTION
**Pull request by Eisei24**
### <samp>🤖 Generated by Copilot at ba02a37</samp>

This pull request adds two new metrics to the validator statistics: the total number and amount of deposits made by each validator. These metrics are stored in the `validator_stats` table, exported in the CSV file, and represented in the `ExporterValidatorStats` struct.

### CODE REVIEW

**Comment by Eisei24** [comment]
This PR prepares everything for the reexport but BIDS-2769 has to be done first to get rid of day -1 which contains deposits.
The reexport of total stats only handles days >= 0.

File: db/migrations/tbd_add_total_deposits.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ALTER TABLE validator_stats ADD COLUMN IF NOT EXISTS deposits_total INT;
+ALTER TABLE validator_stats ADD COLUMN IF NOT EXISTS deposits_amount_total BIGINT;
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+SELECT 'down SQL query - remove total deposits count and amount columns from stats';
+ALTER TABLE validator_stats DROP COLUMN IF EXISTS deposits_total;
+ALTER TABLE validator_stats DROP COLUMN IF EXISTS deposits_amount_total;
+-- +goose StatementEnd
```
Comment (LuccaBitfly) [inline db/migrations/tbd_add_total_deposits.sql:13]: image
I think this means that file should end with newline?

**Review by LuccaBitfly — APPROVED** [review]
utAck

File: db/migrations/tbd_add_total_deposits.sql (hunk 1 above)
Comment (Eisei24) [reply db/migrations/tbd_add_total_deposits.sql:13]: We have some migration files that don't end on a newline but since most of them do I added it.

**Pull request merged**